docs/elements/hx-file-tile/index.html

```diff
@@ -0,0 +1,90 @@
+---
+title: <hx-file-tile>
+also:
+  components/file-tiles: File Tiles
+  components/icons: Icons
+  elements/hx-file-icon: <hx-file-icon>
+  elements/hx-icon: <hx-icon>
+---
+{% extends 'element.njk' %}
+{% block content %}
+
+  <section>
+    <p>
+      The custom <code>{{page.title}}</code> element is compromised of an file icon,
+      file name, file size, and a delete button.
+    </p>
+
+    <hx-dl class="hxBox-md metadata">
+      <hx-def>
+        <hx-dt>Permitted Parents</hx-dt>
+        <hx-dd>any</hx-dd>
+      </hx-def>
+      <hx-def>
+        <hx-dt>Permitted Children</hx-dt>
+        <hx-dd>phrasing content</hx-dd>
+      </hx-def>
+      <hx-def>
+        <hx-dt>Events</hx-dt>
+        <hx-dd>
+          Any of the following:
+          <ul>
+            <li>
+              <code>delete</code> - User clicks "X" to delete the file tile.
+            </li>
+          </ul>
+        </hx-dd>
+      </hx-def>
+    </hx-dl>
+  </section>
+
+  <section>
+    <h2 id="methods">Methods</h2>
+    <dl>
+      <dt>delete()</dt>
+      <dd>Deletes the file tile.</dd>
+    </dl>
+  </section>
+{% endblock %}
+
+{% block attributes %}
+  <dl>
+    <dt>name {String} <i>(optional)</i></dt>
+    <dd>
+      Sets the text of the name.
+      If absent or blank, the name will not be displayed.
+    </dd>
+
+    <dt>href {String} <i>(optional)</i></dt>
+    <dd>
+      Sets the path of the href.
+      if absent, file name will not be a clickable link.
+    </dd>
```

Comment:
Add note that this should only be set if in "default" state (file exists on server).